src/serializer/utils.js

```diff
@@ -72,6 +72,9 @@ export function commonAncestorOf<T>(node1: void | T, node2: void | T, getParent:
     invariant(n2 !== undefined);
     n2 = getParent(n2);
   }
+  if (!n1) {
```

Comment:
This seems very wrong, possibly there to hide some other conceptual issue?

Reply:
Yep, it's wrong but get's the test case in the issue working correctly. Maybe you could spend some time this week to help me work out how this should be done better?

Reply:
Can you also undo this change?